python-opendata/opendata/download.py

```diff
@@ -0,0 +1,85 @@
+from collections import namedtuple
+import glob
+from os import listdir, path, remove
+from shutil import rmtree
+import zipfile
+
+from osfclient import api, cli
+import pandas as pd
+
+
+PROJECT_ID = '6hfpz'
+
+
+Args = namedtuple(
+    'Args',
+    ['project', 'remote', 'local', 'username', 'password', 'force', 'output'],
+)
+
+
+class OpenDataClient:
+    def __init__(self, project_id=PROJECT_ID):
+        self.args = Args(
+            project=project_id,
+            remote=None,
+            local=None,
+            username=None,
+            password=None,
+            force=False,
+            output=None,
+        )
+        self.osf = api.OSF()
+        self.project = self.osf.project(project_id)
+        self.store = next(self.project.storages)
+
+    def download_athlete(self, athlete_id):
```

Comment:
It could be nice to accept a list of id probably

Reply:
That is handled by `opendata.download.download()` (which uses `OpenDataClient.download_athlete`). My idea was to avoid that the end users needs to handle this class and to just offer some easy to use methods (à la requests). After all, the goal of this library is to make using the data from OpenData as easy as possible.